app/helpers/navigation_items.rb

```diff
@@ -74,7 +74,15 @@ def for_provider_account_nav(current_provider_user, current_controller, performi
       items = []
 
       unless performing_setup
-        items << NavigationItem.new(t('page_titles.provider.account'), provider_interface_account_path, active?(current_controller, %w[account profile provider_users organisations provider_relationship_permissions]))
+        if FeatureFlag.active?(:accredited_provider_setting_permissions)
+          if current_provider_user.authorisation.can_manage_users_or_organisations_for_at_least_one_setup_provider?
+            items << NavigationItem.new(t('page_titles.provider.organisation_settings'), provider_interface_organisation_settings_path, active?(current_controller, %w[organisations provider_users]))
```

Comment:
We might need to tweak the markup here as the navigation items is wrapping down a line

![Screenshot 2021-07-07 at 11 18 04](https://user-images.githubusercontent.com/2732945/124743295-82669400-df15-11eb-8379-737eb91e4c93.png)

Reply:
issue if signing in from support, not for providers

Reply:
This doesn't happen unless you're impersonating a provider user from support. Our design prototype wraps the navbar content in the same way, which looks better for normal users (showing Sign out instead of Return to support).